drivers/serial/uart_nrfx_uart.c

```diff
@@ -387,14 +427,21 @@ static int uart_nrfx_init(struct device *dev)
 	/* Enable receiver and transmitter */
 	nrf_uart_enable(NRF_UART0);
 
-	nrf_uart_event_clear(NRF_UART0, NRF_UART_EVENT_TXDRDY);
+	event_txdrdy_clear();
 	nrf_uart_event_clear(NRF_UART0, NRF_UART_EVENT_RXDRDY);
 
 	nrf_uart_task_trigger(NRF_UART0, NRF_UART_TASK_STARTTX);
 	nrf_uart_task_trigger(NRF_UART0, NRF_UART_TASK_STARTRX);
 
 #ifdef CONFIG_UART_INTERRUPT_DRIVEN
 
+	/* Setting flag emulating hardware event informing that UART is ready
+	 * to send data.
+	 * Once first byte is send out readiness status will be provided by
+	 * hardware event: NRF_UART_EVENT_TXDRDY.
+	 */
```

Comment:
What about writing just "Simulate that the TXDRDY event is set, so that the transmitter status is indicated correctly."?

Reply:
done